src/server/app.js

```diff
@@ -43,6 +44,22 @@ app.get('/developers', (req, res, next) => {
   res.status(200).send(page)
 })
 
+//show a selection of unverified submittions
+app.get('/verify', (req, res, next) => {
+  //todo: redirect if the user does not have the proper permissions
+  let submissions=[{
```

Comment:
I did a similar thing for the code review, but this hardcoded list of submissions should be eventually be retrievable from some kind of API call. I think we could create database access functions in a separate file that can be used here to grab the submission list.